lib/cog/commands/help.ex

```diff
@@ -45,28 +43,35 @@ defmodule Cog.Commands.Help do
           nil ->
             {:error, "Bundle #{inspect(bundle_name)} not found"}
           %BundleVersion{} = bundle_version ->
-            body = bundle_version
-            |> Repo.preload(:bundle)
-            |> BundleFormatter.format
-
-            {:ok, body}
+            bundle_version = Repo.preload(bundle_version, :bundle)
+            rendered = Cog.V1.BundleVersionView.render("bundle_version.json", %{bundle_version: bundle_version})
+            {:ok, {:bundle, rendered}}
         end
       [bundle_name, command_name] ->
         full_command_name = bundle_name <> ":" <> command_name
 
         command_version = full_command_name
         |> Commands.with_status_by_any_name
-        |> Repo.preload([:bundle_version, options: :option_type])
+        |> Repo.preload([:command, :bundle_version, options: :option_type])
```

Comment:
Hrm. Always doing a bunch of preloads sounds like a bad idea to me, since it makes the repo call less reusable in other cases where we don't want to preload. Would a chainable `Commands.preloads_for_help` function be a good compromise?